### PR TITLE
White bg/black text, padding on note cell, toolbar subtitle smaller font size

### LINKED ISSUE #55: Pure black and white in note list
To match the composer, and other clients, we should make the bg here white and the title/text black. Date can remain a little faded.

screen shot 2017-02-19 at 10 16 30 am

### PR DESCRIPTION
**Pull request by mobitar**
Fixes #55

First time with Android code, it deleted a few files for some reason.

### CODE REVIEW

**Comment by nbau21** [comment]
Thoughts on untracking the intellij (.idea) files? Are they actually needed upon pulling the repo? @deftelf

File: app/src/main/res/values-v24/styles.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mobitar) [inline app/src/main/res/values-v24/styles.xml]: Not sure why it created this file.

Comment (deftelf) [inline app/src/main/res/values-v24/styles.xml]: Depends on how you created it - possibly there are different defaults for pre-Nougat subtitle layout_marginTop and titleMarginTop

**Comment by deftelf** [comment]
@nbau21 A hot mess is what it is... https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems
I think probably the best thing is to untrack them and let new imports create their own - theoretically you should only need the gradle file to create them. I will do so.

**Comment by deftelf** [comment]
@mobitar I have merged this with a couple of tweaks, removed an unneeded file and fixed some Android Studio config stuff that is probably causing more harm than good.

**Pull request merged**